distribution/docs/src/main/resources/content/_configuring/configuring-identity-node.adoc

```diff
@@ -67,7 +67,7 @@ a|click *Add* button to add associations
 
 |===
 
-.[[service_bindings]]Services
+.[[_service_bindings]]Services
```

Comment:
❓ Why no prefix here?

Reply:
these anchors don't need the prefix because they are marking the destination, and won't change. It's the `<<_linking>>` ones that need to know which file they are referencing